### ISSUE TITLE
Enabling grid view degrades video quality in a 1:1 scenario

### ISSUE THREAD

**Issue by sagz** [Fugiman/google-meet-grid-view#124]
When Grid view is disabled, I see recipient in 720p (~3 mbps bitrate):
Screenshot of quality:
![720p meet](https://user-images.githubusercontent.com/964843/82479577-70166600-9a87-11ea-8ae4-895beccb6c18.png)

Screenshot with Grid view enabled (no advanced settings changed, v1.30):
![gridviewenabled](https://user-images.githubusercontent.com/964843/82479838-c5527780-9a87-11ea-986d-a456e6723b18.png)

I also have a screencast showing me toggling it on and off. I'm happy to share it privately due to sensitive information in it.

**Expected behavior**
Quality shouldn't degrade.

**Software used:**
 - OS [e.g. Windows 10]: 10.15.4 (19E287)
 - Browser [e.g. Chrome, Firefox]: Firefox 78.0a1 (2020-05-20) (64-bit)
 - Extension [e.g. Official Chrome Extension, Tampermonkey]: Official extension from AMO
 - Grid View Version [e.g. 1.19, 1.23]:  1.30

**Comment by sagz** [comment]
I quantified the resolution using [this addon](https://blog.mozilla.org/webrtc/new-tool-debugging-webrtc/)
[Here is a video showing the transition happening](https://github.com/Fugiman/google-meet-grid-view/files/4658958/Shot.2020-05-20.at.11.49.43.mp4.zip)


Grid view switches resolution to 360p. Without grid-view enabled it goes up to 720p

**Comment by sagz** [comment]
Any idea what's causing this @Fugiman ?

Maybe even with auto-enable, we can have grid view disabled for 1:1 (or even 1:2) scenarios and let Meet do it's native thing?

**Comment by Fugiman** [comment]
No. I verified that this does happen but don't know why. I'll look into it again once I have time but for now I recommend just disabling grid view in 1:1s where quality is a concern.
